.devcontainer/config/sailbot_workspace.code-workspace

```diff
@@ -41,36 +41,13 @@
 		"python.formatting.autopep8Args": [
 			"--max-line-length=120"
 		],
-		"C_Cpp.default.intelliSenseMode": "linux-gcc-x86",
-		"C_Cpp.formatting": "Disabled",
-		"uncrustify.configPath.linux": "/opt/ros/eloquent/lib/python3.6/site-packages/ament_uncrustify/configuration/ament_code_style.cfg",
-		"[cpp]": {
-			"editor.defaultFormatter": "zachflower.uncrustify"
-		},
 		"search.exclude": {
 			"**/build": true,
 			"**/install": true,
 			"**/log": true
 		},
 		"workbench.colorCustomizations": {
-			"activityBar.activeBackground": "#19488d",
-			"activityBar.activeBorder": "#dd4181",
-			"activityBar.background": "#19488d",
-			"activityBar.foreground": "#e7e7e7",
-			"activityBar.inactiveForeground": "#e7e7e799",
-			"activityBarBadge.background": "#dd4181",
-			"activityBarBadge.foreground": "#e7e7e7",
-			"sash.hoverBorder": "#19488d",
-			"statusBar.background": "#113262",
-			"statusBar.foreground": "#e7e7e7",
-			"statusBarItem.hoverBackground": "#19488d",
-			"statusBarItem.remoteBackground": "#113262",
-			"statusBarItem.remoteForeground": "#e7e7e7",
-			"titleBar.activeBackground": "#113262",
-			"titleBar.activeForeground": "#e7e7e7",
-			"titleBar.inactiveBackground": "#11326299",
-			"titleBar.inactiveForeground": "#e7e7e799",
-			"commandCenter.border": "#e7e7e799"
```

Comment:
You removed the border colours again, I think you have to manually uninstall peacock to stop this from happening.

Reply:
Weird, I don't have peacock installed at all.